ci/Jenkinsfile-pyp

```diff
@@ -0,0 +1,205 @@
+
+                            # Bump version
+                            ${WORKSPACE}/env/bin/bumpversion --commit ${version_increment}
+                            git push origin next
+                        '''
+                    }
+                }
+            }
+        }
+
+        stage("Add tag") {
+            when {
+                allOf {
+                    expression { "${run_job}" == "true" }
+                    expression { "${release_type}" == "stable" }
+                }
+            }
+            steps{
+                dir('inmanta-sphinx') {
+                    withCredentials([string(credentialsId: 'fff7ef7e-cb20-4fb2-a93b-c5139463c6bf', variable: 'GITHUB_TOKEN')]) {
+                        sh '''
+                            export GIT_ASKPASS="${WORKSPACE}/ASK_PASS"
+                            current_version=$(sed -n -E 's/current_version[[:space:]]*=[[:space:]]*([\\digit.]*)/\\1/p' .bumpversion.cfg)
+                            tag="v${current_version}"
+                            git tag -a -m "Release: ${current_version}" ${tag}
+                            git push origin ${tag}
+                        '''
+                    }
+                }
+            }
+        }
+
+        stage("Build dist") {
+            when {
+                expression { "${run_job}" == "true" }
+                anyOf {
+                    expression { "${release_type}" == "stable" }
+                    expression { "${release_type}" == "dev" }
+                }
+            }
+            steps {
+                dir('inmanta-sphinx') {
+                    sh '''
+                        ${WORKSPACE}/env/bin/python3 -m pip install -U twine pip bumpversion
+                        if [ "${release_type}" == "stable" ]; then
+                            ${WORKSPACE}/env/bin/python3 setup.py egg_info -Db "" sdist
+                        else
+                            ${WORKSPACE}/env/bin/python3 setup.py sdist
+                        fi
+                    '''
+                }
+            }
+        }
+
+        stage("Publish to PyPi") {
+            when {
+                allOf {
+                    expression { "${run_job}" == "true" }
+                    expression { "${upload}" == "true" }
+                    expression { "${release_type}" == "stable" }
+                }
+            }
+            steps {
+                withCredentials([
+                    usernamePassword(
+                        credentialsId: 'inmantaci_on_pypi',
+                        passwordVariable: 'TWINE_PASSWORD',
+                        usernameVariable: 'TWINE_USERNAME'
+                    ),
+                    string(credentialsId: 'package_gpg_passphrase', variable: 'PGP_PASS')
+                ]) {
+                    dir('inmanta-sphinx') {
+                        sh '''
+                            ${WORKSPACE}/env/bin/python3 -m twine upload dist/*.tar.gz
```

Comment:
does this also sign? That used to be a separated gpg command?

Reply:
I don't know, but the [original job](https://jenkins.inmanta.com/job/releases/job/python-packages/job/sphinx-inmanta-pypi-release/) didn't contain any other steps.

Reply:
see here: https://github.com/inmanta/irt/blob/master/ci/inmanta-core/Jenkinsfile-pyp#L224

Reply:
@arnaudsjs Should I add a step like this? As mentioned above it isn't present in the current inmanta-sphinx release job but as Wouter shows it is present in the core release.

Reply:
Yes. Signing packages is always good practice. I assume this step was forgotten during the initial setup of the release job.